### DISCUSSION TITLE
Donate Methods

### DISCUSSION THREAD

**Discussion by ShreyderD** [jgraph/drawio#1909] [Ideas]
Hi, I'm wondering is there any way to donate using crypto?
Abonnement donations are no good choice for me - I don't want to keep in mind all the Abonnements...

**Comment by boone711** [comment]
Yes, PLEASE give us options for donations. Your project is stunning to say the least and I want to show my appreciation. Compare to clunky, pay-me-hundreds-of-dollars-for-proprietary-blah-blah-blah... Lets root for the good guys
